packages/extension-bubble-menu/src/bubble-menu-plugin.ts

```diff
@@ -90,10 +109,17 @@ export class BubbleMenuView {
   }
 
   update(view: EditorView, oldState?: EditorState) {
+    const { viewSelecting } = this
     const { state, composing } = view
     const { doc, selection } = state
-    const isSame = oldState && oldState.doc.eq(doc) && oldState.selection.eq(selection)
 
+    const hideSelecting = this.hideWhenSelecting && viewSelecting
+    if (!view.hasFocus() || hideSelecting) {
+        this.hide()
```

Comment:
`!view.hasFocus()` ... does this break the ability to tab to the bubble menu? Eg, a bubble menu item is focused via keyboard navigation, but this then hides it, because the editor view doesn't have focus.

Reply:
That is a good question and I'm fairly certain this would break the ability to tab to the bubble menu. I don't have that requirement (now), but I'm sure there are those that do, especially for accessibility. Making any UI plugins is so tricky to handle all the use cases. Any suggestions on how we might solve that issue?